cleanlab_studio/version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.1.18"
+__version__ = "0.1.19"
```

Comment:
Let's do version bumps separately from bug/feature PRs. Also, 0.1.19 is already released, so this will need to be in 0.1.20.

Otherwise, LGTM, feel free to merge after removing this (and feel free to bump version and tag a release after merging this).